coast/TIDEGAUGE.py

```diff
@@ -159,28 +165,28 @@ def read_gesla_header_v3(fn_gesla):
         -------
         dictionary of attributes
         """
-        debug(f'Reading GESLA header from "{fn_gesla}"')
+        debug(f"Reading GESLA header from \"{fn_gesla}\"")
         fid = open(fn_gesla)
```

Comment:
As mentioned previously, we could use Path.read_text here. If the file is expected to be large, then we should just use a with block instead.

Reply:
noted for future optimization requirement, if performance takes a hit.